TerminalDocs/customize-settings/startup.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Windows Terminal Startup Settings
+description: Learn how to customize startup settings within Windows Terminal.
+author: cinnamon-msft
+ms.author: cinnamon
+ms.date: 1/28/2021
+ms.topic: how-to
+ms.localizationpriority: high
+---
+
+# Startup settings in Windows Terminal
+
+The properties listed below affect the entire terminal window, regardless of the profile settings. These should be placed at the root of your settings.json file.
+
+> [!TIP]
+> If you'd like to edit these settings using the settings UI, you will have to add the `"openSettings"` action to your `"actions"` array in order to open it with the command palette or keyboard.
+
+```json
+{ "command": { "action": "openSettings", "target": "settingsUI" }, "keys": "ctrl+shift+s" },
+```
+
+> [!IMPORTANT]
+> The settings UI is only available in [Windows Terminal Preview](https://aka.ms/terminal-preview).
+
+## Default profile
+
+Set the default profile that opens by typing <kbd>ctrl+shift+t</kbd>, typing the key binding assigned to `newTab`, running `wt new-tab` without specifying a profile, or clicking the '+' icon.
+
+**Property name:** `defaultProfile`
+
+**Necessity:** Required
+
+**Accepts:** GUID or profile name as a string
+
+**Default value:** PowerShell's GUID
+
+<br />
+
+___
+
+## Launch on machine startup
+
+When set to `true`, this enables the launch of Windows Terminal at startup. Setting this to `false` will disable the startup task entry. Note: if the Windows Terminal startup task entry is disabled either by org policy or by user action this setting will have no effect.
```

Comment:
```suggestion
When set to `true`, this enables the launch of Windows Terminal at startup. Setting this to `false` will disable the startup task entry.

Note: if the Windows Terminal startup task entry is disabled either by org policy or by user action this setting will have no effect.
```
I know you just copied/pasted this, but, maybe it'd look better if the note was on a separate line? Or in a block? idk.